src/ray/core_worker/core_worker.cc

```diff
@@ -2115,16 +2115,25 @@ Status CoreWorker::CreateActor(const RayFunction &function,
   return Status::OK();
 }
 
+namespace {
+bool MatchBundleResource(const std::string &str) {
+  std::stringstream pattern;
+  pattern << "^" << kBundle_ResourceLabel << "_group_([0-9a-f_]+)";
+  static const std::regex resource_pattern(pattern.str());
+  return std::regex_match(str, resource_pattern);
+}
+}  // namespace
+
 Status CoreWorker::CreatePlacementGroup(
     const PlacementGroupCreationOptions &placement_group_creation_options,
     PlacementGroupID *return_placement_group_id) {
   const auto &bundles = placement_group_creation_options.bundles;
   for (const auto &bundle : bundles) {
     for (const auto &resource : bundle) {
-      if (resource.first == kBundle_ResourceLabel) {
+      if (MatchBundleResource(resource.first)) {
```

Comment:
At this point, the resource name is still the original name, hasn't been rewritten/formatted to the pg resource name so we should just do the string comparison.